src/components/v2/VaultItemUi/index.tsx

```diff
@@ -0,0 +1,132 @@
+// VaultItemUi
+/** @jsxImportSource @emotion/react */
+import React, { useMemo } from 'react';
+import Paper from '@mui/material/Paper';
+import Typography from '@mui/material/Typography';
+
+import { useTranslation } from 'translation';
+import { formatCommaThousandsPeriodDecimal, formatToReadablePercentage } from 'utilities/common';
+import { Icon } from '../Icon';
+import { Button } from '../Button';
+import { useStyles } from './styles';
+
+type VaultAssetType = 'vai' | 'xvs' | 'vrt';
+
+export interface IVaultItemUiProps {
+  stakeTokenName: VaultAssetType;
+  earnTokenName: VaultAssetType;
+  rewardValue?: number;
+  stakingValue: number;
+  stakingAprValue: number;
+  dailyEmissionValue: number;
+  totalStakedValue: number;
```

Comment:
To be consistent with the rest of the app:
- values should be suffixed with their unit
- wei values need to use `BigNumber`
- I think `stakingValue` should be more descriptive to explain that it corresponds to the amount staked by the user

```suggestion
  rewardWei?: BigNumber;
  userStakedWei: BigNumber;
  stakingAprPercentage: number;
  dailyEmissionWei: BigNumber;
  totalStakedWei: BigNumber;
```

Reply:
done